src/app/components/Filters/SelectedFilters.jsx

```diff
@@ -169,10 +170,13 @@ class SelectedFilters extends React.Component {
       }
     });
 
+
     if (!filtersToRender.length && !datesToRender.length) {
       return null;
     }
 
+    console.log('filtersToRender: ', filtersToRender);
```

Comment:
We can remove the console log when not needed anymore

Reply:
Thanks, good catch